packages/typography/src/styled/StyledCode.ts

```diff
@@ -29,14 +52,15 @@ const colorStyles = (props: IStyledCodeProps & ThemeProps<DefaultTheme>) => {
 };
 
 interface IStyledCodeProps extends Omit<IStyledFontProps, 'size'> {
-  hue?: string;
+  hue?: ICodeProps['hue'];
   size?: ICodeProps['size'];
 }
 
 export const StyledCode = styled(StyledFont as 'code').attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
-  as: 'code'
+  as: 'code',
+  isMonospace: true
```

Comment:
`isMonospace: true` is required by `StyledFont` to apply the correct styles. 

Defining it as a `defaultProp` didn't make that obvious.

This pattern aligns with other components. For ex:

https://github.com/zendeskgarden/react-components/blob/cffdb06680e0a63bd8faad7cc62085ec83b2f835/packages/buttons/src/styled/StyledAnchor.ts#L18-L25